flake.nix

```diff
@@ -2,14 +2,15 @@
   description = "Flake to develop the CDapp using nix(OS)";
```

Comment:
What does this file / util actually do ?

Reply:
This one is like an nvmrc for nixOS. If you don't want it here, I can remove it.

Reply:
No, leave it in. I was just unsure of what it does.